pkgs/by-name/li/linkwarden/package.nix

```diff
@@ -0,0 +1,155 @@
+{
+  lib,
+  stdenvNoCC,
+  buildNpmPackage,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  makeWrapper,
+  nixosTests,
+  yarnBuildHook,
+  yarnConfigHook,
+  # dependencies
+  bash,
+  monolith,
+  nodejs,
+  openssl,
+  playwright-driver,
+  prisma,
+  prisma-engines,
+}:
+
+let
+  bcrypt = buildNpmPackage rec {
+    pname = "bcrypt";
+    version = "5.1.1";
+
+    src = fetchFromGitHub {
+      owner = "kelektiv";
+      repo = "node.bcrypt.js";
+      rev = "v${version}";
+      hash = "sha256-mgfYEgvgC5JwgUhU8Kn/f1D7n9ljnIODkKotEcxQnDQ=";
+    };
+
+    npmDepsHash = "sha256-CPXZ/yLEjTBIyTPVrgCvb+UGZJ6yRZUJOvBSZpLSABY=";
+
+    npmBuildScript = "install";
+
+    postInstall = ''
+      cp -r lib $out/lib/node_modules/bcrypt/
+    '';
+  };
+
+  version = "2.8.1";
+  src = stdenvNoCC.mkDerivation {
+    pname = "linkwarden-source";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "linkwarden";
+      repo = "linkwarden";
+      rev = "v${version}";
+      hash = "sha256-jAsv69dO9Mi4i2TuGM14rUU4+42UbrM96YXJunYMJWM=";
```

Comment:
I got `sha256-lQBmtnITz2aY75kC57gsp//lKi34ZAt5bueBCmeMDeg=` while building this package on version 2.8.1

Reply:
+1

Reply:
No idea how that happened, fixed now

Reply:
The tag was moved from linkwarden/linkwarden@043589b3015803e195a87a2747540da95dc3f377 to linkwarden/linkwarden@91494b0188434982c5d9ebc201e73a17bdaed33b :unamused: